build.gradle

```diff
@@ -123,16 +123,16 @@ dependencies {
     // in the war/jar. When a war is loaded, the jars in the provided folder are not loaded.
     providedCompile 'org.springframework.boot:spring-boot-starter-jetty:1.3.1.RELEASE'
     // Jcloud dependencies
-    compile ('org.apache.jclouds:jclouds-compute:2.0.1'){
+    compile ('org.apache.stratos:jclouds-compute:1.9.1'){
```

Comment:
Yeah I remember that `stratos` group name, its strange that it changed from `stratos` to `jclouds` only for jclouds-compute.